doc/examples/ml/hyperparameter/README.md

```diff
@@ -0,0 +1,166 @@
+$ pachctl create-pipeline -f select.json
+```
+
+Once the pipelines are up an running you should be able to see their corresponding workers in kubernetes:
+
+```sh
+$ kubectl get pods
+NAME                       READY     STATUS    RESTARTS   AGE
+etcd-2142892294-nzb39      1/1       Running   0          1h
+pachd-776177201-ww198      1/1       Running   0          1h
+pipeline-model-v1-htphr    2/2       Running   0          7m
+pipeline-select-v1-240bt   2/2       Running   0          7m
+pipeline-split-v1-9lwkn    2/2       Running   0          7m
+pipeline-test-v1-h47lw     2/2       Running   0          7m
+```
+
+And, after waiting a few minutes, you should see the successful jobs that did our distributed hyperparameter tuning:
+
+```sh
+$ pachctl list-job
+ID                                   OUTPUT COMMIT                           STARTED            DURATION           RESTART PROGRESS    DL       UL       STATE            
+e2b75a61-13e2-4067-88b7-adec4d32f830 select/f38eae7cea574fc6a90adda706d4714e 18 seconds ago     Less than a second 0       1 + 0 / 1   243.2KiB 82.3KiB  success 
+4116af2b-efa5-405e-ba04-f850a656e25d test/1e379911118c4492932a2dd9eb198e9a   About a minute ago About a minute     0       77 + 0 / 77 400.3KiB 924B     success 
+f628028e-2c88-439e-8738-823fe0441e1b model/6a877b93e3e2445e92a11af8bde6dddf  3 minutes ago      About a minute     0       77 + 0 / 77 635.1KiB 242.3KiB success 
+a2ba2024-db12-4a78-9383-82adba5a4c3d split/04955ad7fda64a66820db5578478c1d6  5 minutes ago      Less than a second 0       1 + 0 / 1   10.29KiB 10.29KiB success
+```
+
+## Looking at the results
+
+If we look at the models that were trained based on our training data, we will see one model for each of the combinations of *C* and *Gamma* parameters:
+
+```sh
+$ pachctl list-file model master
+NAME                        TYPE                SIZE                
+model_C0.031_G0.001.pkl     file                6.908KiB            
+model_C0.031_G0.004.pkl     file                6.908KiB            
+model_C0.031_G0.016.pkl     file                6.908KiB            
+model_C0.031_G0.063.pkl     file                6.908KiB            
+model_C0.031_G0.25.pkl      file                6.908KiB            
+model_C0.031_G1.0.pkl       file                6.908KiB            
+model_C0.031_G4.0.pkl       file                6.908KiB            
+model_C0.125_G0.001.pkl     file                4.85KiB             
+model_C0.125_G0.004.pkl     file                4.85KiB             
+model_C0.125_G0.016.pkl     file                4.85KiB             
+model_C0.125_G0.063.pkl     file                4.85KiB             
+model_C0.125_G0.25.pkl      file                4.85KiB             
+model_C0.125_G1.0.pkl       file                4.85KiB
+etc...
+``` 
+
+There should be 77 of these models:
+
+```sh
+$ pachctl list-file model master | wc -l
+78
+```
+
+But not all of these models are ideal for making our predictions.  
```

Comment:
Is there a 1-2 sentence conclusion we can make here?

For example, "if we look at the final output of the select stage, we can see the modelXYZ using these parameter was actually the best model."

I would also add a sentence or two on generalizing this. e.g.T"he DAG pattern we've built here is very easy to generalize for any sort of exploration of a parameter space. As long as you break up your parameters into individual files as we've shown, you can test the whole parameter space in a massively distributed way and simply pick out the best results."

Reply:
Yeah, I'm glad you caught this. There was actually supposed to be something else here, that I forgot to copy over. Moreover, when I tried to re-run this and recreate I got some weirdness, so dealing with that. I will update here shortly.